src/Allocation.jl

```diff
@@ -24,10 +24,8 @@ include("lib/datastore.jl")
 
     # Extracted water (1000 m3) to be set by optimisation - super source represents failure.
     waterfromgw = Parameter(index=[regions, time], unit="1000 m^3")
-    waterfromreservoir = Parameter(index=[regions,time], unit="1000 m^3")
```

Comment:
Don't we need this for the simulation?  And it should probably end up in the return flows as well.  I see that it's currently not added in, but the other sources of water are, so that's probably a mistake.